src/test.js

```diff
@@ -258,6 +258,7 @@ Test.prototype = {
 		}
 
 		emit( "testEnd", this.testReport.end( true ) );
+		this.testReport.slimAssertions();
```

Comment:
Seems a bit weird to me that the `testEnd` event gets the full assertions, but the `testDone` logging callback does not. Can you explain your reasoning here?